### PR TITLE
20191102 csv totals

### PR DESCRIPTION
**Pull request by ScottMcNay**
Need better way to strip currency symbols

### CODE REVIEW

File: src/js/csv.js [hunk 1]
```diff
@@ -9,14 +9,30 @@ function download(table) {
...
             let cells = rows[i].cells;
             let cell_array = [];
             for(let j=0; j<cells.length; ++j) {
-                cell_array.push(cells[j].textContent);
+                cell_array.push(cells[j].textContent.replace('$',''));
             }
             result.push(cell_array);
         }
+        const cells = [
+            '=SUBTOTAL(103,A2:A{LAST}) & " items"',
```
Comment (philipmulcahy) [inline src/js/csv.js]: not everyone who is using the csv is going to put it into excel :(

**Comment by ScottMcNay** [comment]
I tested with Google Sheets and LibreOffice Calc; that's what the second
commit is for (lemme double-check after breakfast that I actually sent the
second commit). LibreOffice is a fork of OpenOffice, so OpenOffice should
work also.

I didn't try with WordPerfect Office. I know they used to have incompatible
formatting for formulas years and years ago (semicolons instead of commas),
but the webpage now claims to be 100% compatible.

I used the Subtotal function because it respects filtering, else I would
have just done the calculation inline. Could always ask the user.

I had to strip the currency symbols because LibreOffice pulled them in as a
string value instead of a currency type like Excel does. Just need a better
way of filtering different currency symbols.

--Scott.

**Comment by philipmulcahy** [comment]
how would you feel about having some kind of mode setting (perhaps a checkbox or radio button on the popup) that gets saved in a cookie and is used to tell the csv func whether to include the SS specific stuff (which would definitely be useful) or leave it out (also useful)?
I was driving home today thinking about how to do this cleanly.
I reckon you could put the cookie save and read stuff in the csv.js file and import it from both the popup and from the injected side (it’s already imported there) - that would give you persistence without needing to add any more messaging.

**Comment by ScottMcNay** [comment]
Replaced by a newer PR.

**Pull request closed without merging**